client/api_bucket.go

```diff
@@ -154,7 +154,13 @@ func (c *client) CreateBucket(ctx context.Context, bucketName string, primaryAdd
 		return "", err
 	}
 
-	return resp.TxResponse.TxHash, err
+	txnHash := resp.TxResponse.TxHash
```

Comment:
It is recommended to add is_async field to the option, which is false by default.